p2p/pubsub/pubsub.go

```diff
@@ -139,15 +141,40 @@ const (
 	ValidationReject = pubsub.ValidationReject
 )
```

Comment:
i think we should remove all aliases above

Reply:
Do you know the reason for adding them in the first place, they seemed kind of unnecessary to me?

Reply:
i defined aliases so that libp2p is not imported directly across project codebase, it is easier to prevent misuse this way